projects/storefrontapp-e2e-cypress/cypress/integration/regression/site-context/language/language-my-account-pages.e2e-spec.ts

```diff
@@ -111,9 +111,17 @@ describe('Language switch - my-account pages', () => {
         siteContextSelector.LANGUAGE_LABEL
       );
 
-      cy.get('cx-update-profile-form select')
-        .select(deutschName)
-        .should('have.value', 'mr');
+      cy.get('cx-update-profile-form').within(() => {
+        cy.get('[formcontrolname="titleCode"]')
+          .click()
+          .within(() => {
+            cy.get('.ng-dropdown-panel .ng-option:nth-child(2)').click();
+          });
```

Comment:
I believe that line 114-119 can be shortened to this:
```
cy.get('[formcontrolname="titleCode"]').ngSelect('Mr.');
```

Reply:
Good catch. Googled for cypress and ngSelect any docs about it.

I also included changes to update-profile.e2e-spec.ts since I used the same method there.